aiidalab/git_util.py

```diff
@@ -223,10 +237,10 @@ def get_current_branch(self):
             )
         return branch.strip()
 
-    def get_commit_for_tag(self, tag):
+    def get_commit_for_tag(self, tag: str):  # type: ignore
```

Comment:
Not sure why here the type ignore required, this function return value, possible to add the type for `self.get_peeled`?

Reply:
I think `get_peeled` is inherited from `dulwich`. I've added `Any` as return type. Looks like `decode` function returns Any, so not sure we can do better